src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -285,7 +286,16 @@ public Parent getNode() {
 
     @FXML
     private void addNewFile() {
-        viewModel.addNewFile();
+        LinkedFileDialogView controller = new LinkedFileDialogView();
+
+        // Show the dialog using DialogService
+        LinkedFile newLinkedFile = dialogService.showCustomDialogAndWait(controller).orElse(null);
+
+        // Handle the result if the user clicked OK or Add
+        if (newLinkedFile != null) {
+            // Add the new file to your list or handle it as needed
```

Comment:
Which AI assistant did you use? I want to learn about their capabiltiies.

Reply:
@koppor I generally use ChatGPT